src/Sentry/Protocol/Envelopes/Envelope.cs

```diff
@@ -325,6 +326,34 @@ public static Envelope FromTransaction(Transaction transaction)
         return new Envelope(eventId, header, items);
     }
 
+    /// <summary>
+    /// Creates an envelope that contains one or more <see cref="CodeLocations"/>
+    /// </summary>
+    internal static Envelope FromCodeLocations(CodeLocations codeLocations)
+    {
+        var header = DefaultHeader;
+
+        List<EnvelopeItem> items = [ EnvelopeItem.FromCodeLocations(codeLocations) ];
```

Comment:
Does this create a List with capacity=1 ? If not the optimization I mentioned before didn't make it through (though this syntax is much nicer)

Reply:
Apparently not... just tried it out and the deafult capacity is 4. I'll use the longer syntax that allocates less memory though.